lms/djangoapps/program_enrollments/api/linking.py

```diff
@@ -169,6 +170,7 @@ def link_program_enrollment_to_lms_user(program_enrollment, user):
                 user, pce.course_key, pce.status
             )
             pce.save()
+            _realize_course_access_roles(user, pce)
```

Comment:
Putting this here instead of `link_program_enrollments` has the benefit that it will also be called as part of [matriculate_learner](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/program_enrollments/signals.py#L47). Otherwise, this functionality would only be exposed via the the Support Tool.